app/src/app/[lng]/admin/page.tsx

```diff
@@ -90,6 +101,9 @@ const AdminPage = ({ params: { lng } }: { params: { lng: string } }) => {
   const [createOrganizationInvite, { isLoading: isInviteLoading }] =
     api.useCreateOrganizationInviteMutation();
 
+  const [updateOrganizationActiveStatus, { isLoading: isUpdatingStatus }] =
+    api.useMutateAccountFrozenStatusMutation();
```

Comment:
Can you change either the name of the API hook or the name you are using for the function here? So both use either active or frozen in their terminology (as these terms are opposites).